PCPTool.v11/exe/SDKSample.cpp

```diff
@@ -3414,8 +3415,16 @@ or in the machine context.
                                             &hKey,
                                             pKeyName->pszName,
                                             0,
-                                            0))))
+                    dwFlags[n] & NCRYPT_MACHINE_KEY_FLAG))))
                 {
+                    wprintf(L"Could not open key: %s (0x%08lx)\n", pKeyName->pszName, hr);
+                    if (pKeyName != NULL)
+                    {
+                        NCryptFreeBuffer(pKeyName);
+                        pKeyName = NULL;
+                    }
+                    hr = S_OK;
+                    continue;
                     goto Cleanup;
```

Comment:
the ```goto``` statement can be deleted.

Reply:
Thank you for spotting it. I have removed it!